src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js

```diff
@@ -6,7 +6,7 @@ import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
 import WarningAmberOutlinedIcon from '@mui/icons-material/WarningAmberOutlined';
 import { Accordion, AccordionSummary, AccordionDetails, Button, Card, Grid, Paper, Typography } from '@mui/material';
-import ScenarioPowerBiReport from '../ScenarioPowerBiReport';
+import ScenarioChartReport from '../ScenarioChartReport/ScenarioChartReport';
```

Comment:
This import line can be simplified to
`import ScenarioChartReport from '../ScenarioChartReport';`
thanks to the export in the associated _index.js_ file 🙂